doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -41,40 +41,60 @@ data in *folds* that we use for training and testing::
     >>> print(scores)
     [0.93489148580968284, 0.95659432387312182, 0.93989983305509184]
 
-.. currentmodule:: sklearn.cross_validation
+.. currentmodule:: sklearn.model_selection
 
-This is called a :class:`KFold` cross validation
+This is called a :class:`KFold` cross-validation.
 
 .. _cv_generators_tut:
 
 Cross-validation generators
 =============================
 
+Scikit-learn has a collection of classes which can be used to generate lists of
+train/test indices for popular cross-validation strategies.
 
+The classes are data-independent and are initialized only with parameters that
```

Comment:
This sentence is a bit odd to me. If you didn't know the way it worked before, you wouldn't expect it to be data dependent, right? Also it is unclear what this means.